### PR TITLE
Update DALi documents for Tizen 4.0

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### CODE REVIEW

File: docs/application/native/guides/ui/dali/actors-n.md [hunk 1]
```diff
@@ -5,7 +5,9 @@
 
 An actor is the basic component that composes the entire scene. It can have visible (for example, UI components) or invisible (for example, a camera actor or layer) forms.
 
-An actor is also the primary object with which DALi applications interact. Multiple types of event signals provided by actors can be handled in an application through user-defined callback functions.
+An actor is also the primary object with which DALi applications interact. Controls and Layers all derive from Actor.
+
```
Comment (surbhig12) [inline docs/application/native/guides/ui/dali/actors-n.md]: Controls and Layers all derive from Actor.

Please rephrase to:

All Controls and Layers are derived from Actors.

File: docs/application/native/guides/ui/dali/actors-n.md [hunk 2]
```diff
@@ -5,7 +5,9 @@
...
+Multiple types of event signals provided by actors can be handled in an application through user-defined callback functions.
```
Comment (surbhig12) [inline docs/application/native/guides/ui/dali/actors-n.md]: Multiple types of event signals provided by actors can be handled in an application through user-defined callback functions.

Please rephrase to:

In an application, multiple types of event signals provided by actors can be handled through user-defined callback functions.

File: docs/application/native/guides/ui/dali/actors-n.md [hunk 3]
```diff
@@ -21,13 +23,19 @@ An actor has the following concrete types:
 - **Camera actor** determines which view of the whole virtual 3D world is rendered on the screen. By default, it is best suited for 2D applications, so you do not need to care about it in this case. For more information, see the `Dali::CameraActor` class (in [mobile](../../../../../org.tizen.native.mobile.apireference/classDali_1_1CameraActor.html) and [wearable](../../../../../org.tizen.native.wearable.apireference/classDali_1_1CameraActor.html) applications).
 - **Layer** provides a mechanism for overlaying groups of actors on top of each other. For more information, see the `Dali::Layer` class (in [mobile](../../../../../org.tizen.native.mobile.apireference/classDali_1_1Layer.html) and [wearable](../../../../../org.tizen.native.wearable.apireference/classDali_1_1Layer.html) applications).
 
-## Actors and Stage
+## Adding an Actor to the Stage
+
+Stage is a top-level object that represents the entire screen.
+It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure, which means an actor inherits a position relative to its parent, and can be moved in relation to this point.
```
Comment (surbhig12) [inline docs/application/native/guides/ui/dali/actors-n.md]: It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure, which means an actor inherits a position relative to its parent, and can be moved in relation to this point.
<<The last sentence ...moved in relation to this point..is not clear, I have tried to rephrase, please let me know if its technically fine>>

Please rephrase to:

It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure. This implies that an actor inherits a position relative to its parent position and can be moved to another position with respect to its current position.

File: docs/application/native/guides/ui/dali/actors-n.md [hunk 4]
```diff
@@ -21,13 +23,19 @@ An actor has the following concrete types:
...
 
-Stage is a top-level object that represents the entire screen. It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure, which means an actor inherits a position relative to its parent, and can be moved in relation to this point.
+The stage instance is a singleton object (the only instance of its class during the lifetime of the program). You can get it using a static function:
```
Comment (surbhig12) [inline docs/application/native/guides/ui/dali/actors-n.md]: You can get it using a static function:

Please rephrase to:

You can get a stage function using the following static function:

**Comment by Seoyeon2Kim** [comment]
I had a mistake :( .. Reopened the commit.

File: docs/application/native/guides/ui/dali/actors-n.md (hunk 1 above)
Comment (Seoyeon2Kim) [reply docs/application/native/guides/ui/dali/actors-n.md]: Okay, changed the description.

File: docs/application/native/guides/ui/dali/actors-n.md (hunk 2 above)
Comment (Seoyeon2Kim) [reply docs/application/native/guides/ui/dali/actors-n.md]: Okay, changed the description.

File: docs/application/native/guides/ui/dali/imageview-n.md [hunk 5]
```diff
@@ -3,27 +3,80 @@
...
+
+ImageView has the following properties:
+
+| Property  | Type | Description |
+| ------------ | ------------ | ------------ |
+| IMAGE  | Map | map of properties associated with a given image. |
+| PRE_MULTIPLIED_ALPHA | bool | opacity adjusted image. |
+| PIXEL_AREA | Vector4 | sub area of image (Animatable). |
+
+> **Note**
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/dali/imageview-n.md]: It looks like the below contents are all in Note style. (from line 70 PreMultipliedAlpha to line 82). Is it correct?

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**